paramak/shape.py

```diff
@@ -791,7 +795,7 @@ def export_stp(self, filename=None, units='mm', mode='solid'):
                     only accepts 'solid' or 'wire'", self)
 
         if units == 'cm':
-            self._replace(
+            _replace(
```

Comment:
this isn't tested

Reply:
thanks for spotting this, I can add tests for that